charm/templates/snap-build-poller_logrotate.j2

```diff
@@ -0,0 +1,12 @@
+# DO NOT EDIT: Managed by juju charm
+
+{{logs_dir}}/poller.log {
+    missingok
+    notifempty
+
+    delaycompress
+    rotate 30
+    daily
+    dateext
+    compress
+  }
```

Comment:
Might be a good idea for this to have a trailing blank line just in case some version of logrotate gets upset by jinja2 eating the trailing newline.

Also the space before the last closing brace irrationally annoys me. :-)

Reply:
Good point, added the extra newline and dropped the stray spaces. Thanks.